frontend/src/views/CellGuide/components/CellGuideCard/components/common/Table/index.tsx

```diff
@@ -16,6 +16,7 @@ interface TableProps<T> {
     string | React.ReactElement
   >;
   testId?: string;
+  hoverable?: boolean;
```

Comment:
DE results table has a design requirement that rows do not change color on hover.